pulp_container/app/registry_api.py

```diff
@@ -951,13 +993,30 @@ def handle_safe_method(self, request, path, pk):
             try:
                 tag = models.Tag.objects.get(name=pk, pk__in=repository_version.content)
             except models.Tag.DoesNotExist:
-                raise ManifestNotFound(reference=pk)
+                if distribution.remote:
+                    remote = distribution.remote.cast()
+                    repository = distribution.repository.cast()
+                    manifest = self.fetch_manifest(remote, repository_version, repository, pk)
```

Comment:
please add a comment why we fetch here with HEAD ---> to avoid 302---> possible 404

Reply:
Also, I will return 429 from content-app in case we reach the pull limit from the dockerhub registry after issuing a GET request there.